vnext/Shared/OInstance.cpp

```diff
@@ -53,6 +53,10 @@
 #include <ReactCommon/TurboModuleBinding.h>
 #include "ChakraRuntimeHolder.h"
 
+#if (defined(_MSC_VER) && !defined(WINRT))
+using Microsoft::React::WebSocketModule;
+#endif
```

Comment:
/curious why is this necessary in the non winrt case?

Reply:
This class (for now, at least) is Desktop-only.
The Universal solution will fail to build because its projects don't know about it.

Soon enough, I think we want to split (and rename, for the `O` starts for `Office`) this file aprropriately.

Reply:
Capture this in a comment?